dll/directx/wine/d3dcompiler_43/CMakeLists.txt

```diff
@@ -23,12 +21,14 @@ list(APPEND SOURCE
 
 add_library(d3dcompiler_43 SHARED
     ${SOURCE}
+    asmshader.tab.c
+    asmshader.yy.c
     version.rc
     ${CMAKE_CURRENT_BINARY_DIR}/d3dcompiler_43.def)
 
 set_module_type(d3dcompiler_43 win32dll)
 target_link_libraries(d3dcompiler_43 dx10guid uuid wine wpp)
 add_importlibs(d3dcompiler_43 msvcrt kernel32 ntdll)
 add_dependencies(d3dcompiler_43 d3d_idl_headers)
-add_pch(d3dcompiler_43 d3dcompiler_private.h SOURCE)
+add_pch(d3dcompiler_43 d3dcompiler_private.h SOURCE UNITY_DEBUGCHANNELS_COMPAT)
```

Comment:
Hm, when going over it again, maybe `UNITYBUILD_DEBUGCHANNELS_COMPAT` would be even better?